prerequisites.sh

```diff
@@ -39,6 +39,9 @@ if [ -d "$LIBS_DIR" ] && [ "$(ls -A $LIBS_DIR)" ]; then
 fi
 
 # Download libsemigroups
-echo  "Downloading libsemigroups v$VERS into $LIBS_DIR..."
-curl -L -O "https://github.com/libsemigroups/libsemigroups/releases/download/v$VERS/libsemigroups-$VERS.tar.gz"
-tar -xzf "libsemigroups-$VERS.tar.gz" && rm -f "libsemigroups-$VERS.tar.gz" && mv "libsemigroups-$VERS" "$LIBS_DIR"
+echo  "Downloading libsemigroups/libsemigroups/master into $LIBS_DIR..."
```

Comment:
Can this change now be reversed, with the `.LIBSEMIGROUPS_VERSION` file being updated to 2.0.0?

Reply:
Good idea!